src/_skip/Hydrocarbon_Processing_example.ipynb

```diff
@@ -184,28 +153,40 @@
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.VLE_props.octene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.ethane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.propane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.nbutane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.ibutane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.ethylene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.propene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.butene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.pentene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.hexene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.heptene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
-      "2022-04-07 16:02:16 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.Vap_props.octene dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
+      "2022-08-24 10:05:08 [WARNING] idaes.models.properties.modular_properties.pure.Perrys: DEPRECATED - fs.VLE_props.ethane dens_mol_liq_comp_coeff index 'eqn_type' should be specified, defaulting to equation form 1.\n",
```

Comment:
These warnings are easy fixes on the main codebase side, and we may want to address them before merging.